### PR TITLE
[1.7.0] Add cross-link, re-write test docs

### LINKED ISSUE #481: [1.7.0] Document test-case-selector flag for test cli command
### Page link

_No response_

### Description

kyverno/kyverno#3183 needs to be documented.

### Expected behavior

A brief description and few examples about the flag under https://kyverno.io/docs/kyverno-cli/#test

### Slack discussion

_No response_

### LINKED ISSUE #486: [1.7.0] Document the new `--registry` flag in the CLI
### Page link

See https://github.com/kyverno/kyverno/pull/3396

### Description

In https://github.com/kyverno/kyverno/pull/3396 we added a new `--registry` flag which is enabled uses the local docker credentials to auth and access the registries.

### Expected behavior

Document the above.

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by chipzoller**
## Related issue #

Closes #486 
Closes #481 

## Proposed Changes

* Adds cross-link to the other filter
* Rewrite of the `test` CLI docs

Needs cherry pick to 1.7.0
## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

File: content/en/docs/Kyverno CLI/_index.md [hunk 1]
```diff
@@ -682,51 +682,23 @@ summary:
...
 
-```sh
-kyverno test https://github.com/kyverno/policies/pod-security/restricted --git-branch release-1.5
-```
+1. pass: The resource passes the policy definition. For `validate` rules which are written with a `deny` statement, this will not be a possible result. `mutate` rules can declare a pass.
+2. skip: The resource does not meet either the `match` or `exclude` block, or does not pass the `preconditions` statements. For `validate` rules which are written with a `deny` statement, this is a possible result. If a rule contains certain conditional anchors which are not satisfied, the result may also be scored as a `skip`.
+3. fail: The resource does not pass the policy definition. Typically used for `validate` rules with pattern-style policy definitions.
```
Comment (samj1912) [inline content/en/docs/Kyverno CLI/_index.md:695]: It is also possible to add a `warn` state. See https://github.com/kyverno/kyverno/pull/3821

Comment (chipzoller) [reply content/en/docs/Kyverno CLI/_index.md:695]: That's for apply. Does it work for test?

Comment (samj1912) [reply content/en/docs/Kyverno CLI/_index.md:695]: Yup, there is a linked test case for `test` as well.

Comment (samj1912) [reply content/en/docs/Kyverno CLI/_index.md:695]: See https://github.com/moolen/kyverno/blob/f83c398bad078b519f735121cb7b17d11ffad75d/test/cli/test/depecated_apis/kyverno-test.yaml#L16

Comment (chipzoller) [reply content/en/docs/Kyverno CLI/_index.md:695]: Thank you for catching that! This is why I love peer review. Corrected in https://github.com/kyverno/website/pull/556/commits/5fc5ef71aaf27914e8475d4a42a28d901b69df07

**Review by samj1912 — APPROVED** [review]

File: content/en/docs/Kyverno CLI/_index.md [hunk 2]
```diff
@@ -682,51 +682,24 @@ summary:
...
+4. warn: Setting the annotation `policies.kyverno.io/scored` to `"false"` on a resource which would otherwise fail will be considered a `warn`.
```
Comment (samj1912) [inline content/en/docs/Kyverno CLI/_index.md:696]: This also works if the annotation is on the policy I believe

Comment (chipzoller) [reply content/en/docs/Kyverno CLI/_index.md]: Addressed in https://github.com/kyverno/website/pull/556/commits/33a15a238bec82ff6fe51c623dbbd0911cc9adf8

**Review by samj1912 — APPROVED** [review]

**Pull request merged**